internal/handler/command_test.go

```diff
@@ -18,7 +18,7 @@ func TestParseWriteParamsWrongParamName(t *testing.T) {
 
 	_, err := parseWriteParams(profileName, roMap, params)
 
-	if  err == nil {
+	if err == nil {
```

Comment:
whitespace only changes in this file (see above)!

Reply:
white space changed by go fmt